src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_office_excel.g.cs

```diff
@@ -199,7 +199,7 @@ public ClientData(string outerXml)
     }
 
     
-private static readonly ParticleConstraint _constraint = new CompositeParticle(ParticleType.Choice, 0, 0)
+private static readonly CompiledParticle _constraint = new CompositeParticle(ParticleType.Choice, 0, 0)
```

Comment:
Nope - there's a generator (that's closed source at the moment) that an equivalent change has been made so it will regenerate the `.g.cs` the same for subsequent runs.